gulpfile.js

```diff
@@ -0,0 +1,90 @@
+var gulp = require('gulp');
```

Comment:
I see it does the transpile ;)
You still can use relative path for client codes using browserify: http://browserify.org
Eventually we'll deploy client codes and server codes to separated hosts (servers). Please think in that way.